app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ class ApplicationController < ActionController::Base
   private
 
   def display_subject?
-    return true if Octobox.config.fetch_subject
+    return true if Octobox.fetch_subject?
```

Comment:
Moved a number of boolean checks onto the `Octobox` class with a question mark which reference the non-questionmark config options